frontend/src/components/UserSearch.vue

```diff
@@ -28,7 +28,7 @@
         />
       </v-col>
       <v-col class="col-4">
-          <v-btn id="search-button" class="secondary" medium @click.native="searchUser('&search='+userSearchInput)">Search Users</v-btn>
+          <v-btn id="search-button" class="primary" medium @click.native="searchUser('&search='+userSearchInput)">Search Users</v-btn>
```

Comment:
udpated button colour to match bcgov standards 
https://developer.gov.bc.ca/Design-System/Primary-Button